simphony_ui/ui.py

```diff
@@ -110,8 +111,16 @@ class Application(HasStrictTraits):
     )
 
     @on_trait_change('calculation_error_event', dispatch='ui')
-    def show_error(self, msg):
-        error(None, 'Oups ! Something went bad:\n{}'.format(msg), 'Error')
+    def show_error(self, messages):
+        error_msg, tb_msg = messages
+        error(
+            None,
+            'Oups ! Something went bad:\n{}\n\nDetails:\n{}'.format(
```

Comment:
Hum... Are we really talking about the pyface.api.error function ? Because I don't have detail and informative arguments.